requirements/extras.txt

```diff
@@ -1,3 +1,5 @@
-rapidfuzz~=2.0
-ovos-bus-client < 0.1.0
-ovos-config < 0.1.0
\ No newline at end of file
+rapidfuzz~=3.6
+ovos_plugin_manager>=0.0.25a2
+ovos-config>=0.0.12
+ovos-workshop>=0.0.13a22
+ovos-bus-client >=0.0.8a1
```

Comment:
Can these specify stable releases? It can be worked around since these are extras, but better to specify stable dependencies in stable releases so they don't always pull the latest alpha release in the future

Reply:
not sure where these are used, guess we need to check where versions were introduced to see if its safe/what feats this package depends on